app/controllers/admin/upgrade_notices_controller.rb

```diff
@@ -5,6 +5,8 @@ class Admin::UpgradeNoticesController < FrontendController
 
   activate_menu :upgrade_notices
 
+  layout false # ???? Where and when is this rendered
```

Comment:
I figured out that this is used in SaaS, when a tenant has some limitations (some disabled so-called "feature switches").
For example, if the current plan allows only one application per account, clicking on the Create Application button redirects to: `/admin/upgrade_notices/multiple_applications`.

This is how it looks like in production:
![Screenshot from 2023-06-06 00-20-31](https://github.com/3scale/porta/assets/1270328/05f49558-d75e-4293-a515-fadf1778a2d3)

And this is this branch:
![Screenshot from 2023-06-06 00-21-00](https://github.com/3scale/porta/assets/1270328/3c57d502-4ba1-4778-abcf-cf96a6ab389c)

(except that in production there is no "upgrade to plan `X" button, and in dev there is one.

Reply:
Just removing this line is good enough IMO:

![Screenshot from 2023-06-06 00-27-22](https://github.com/3scale/porta/assets/1270328/839567df-b2a7-45f7-a10d-645d3ffb8572)

Reply:
👍🏻